crt_portal/static/sass/styles.scss

```diff
@@ -48,4 +48,4 @@
 @forward "custom/links";
 @forward "custom/modal";
 @forward "custom/print";
-@forward "custom/error";
+@forward "custom/error";
```

Comment:
supernit: [configure your editor to add newlines at EOF](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)